@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fmc/fn_fmcHeadingHold.sqf

```diff
@@ -0,0 +1,72 @@
+params ["_heli", "_deltaTime"];
+#include "\fza_ah64_sfmplus\headers\core.hpp"
+
+private _pidHdg     = _heli getVariable "fza_sfmplus_pid_hdgHold";
+private _pidTrn     = _heli getVariable "fza_sfmplus_pid_trnCoord";
+//_pidTrn set ["kp", KP];
+//_pidTrn set ["ki", KI];
+//_pidTrn set ["kd", KD];
+private _curHdg     = getDir _heli;
+private _desiredHdg = _heli getVariable "fza_ah64_hdgHoldDesiredHdg";
+private _hdgError   = [_curHdg - _desiredHdg] call CBA_fnc_simplifyAngle180;
+private _curVel     = vectorMagnitude [velocityModelSpace _heli # 0, velocityModelSpace _heli # 1];
+private _subMode    = _heli getVariable "fza_ah64_hdgHoldSubMode";
+private _attSubMode = _heli getVariable "fza_ah64_attHoldSubMode";
+private _output     = 0.0;
+//Breakout values expand as the aircraft goes faster to provide good pedal response
+//at a hover. The expanded range is meant to de-sensitize the pedasl in order to 
```

Comment:
Good documentation, please can you fix the typo in "pedal"?

Reply:
Fixed